packaging/debian/prerm

```diff
@@ -19,10 +19,6 @@ set -e
 
 case "$1" in
     remove|upgrade)
```

Comment:
Now that your prerm is essentially empty, you can just delete the file. One will be created by debhelper if needed.

Reply:
You deleted the `postrm.ex`. But you can also delete the `prerm`; it is empty now and serves no purpose.